carddrawer/src/test/java/com/meli/android/carddrawer/format/CardDrawerFontTest.kt

```diff
@@ -2,89 +2,212 @@ package com.meli.android.carddrawer.format
 
 import android.graphics.Typeface
 import com.mercadolibre.android.andesui.font.Font
+import io.mockk.mockk
 import org.junit.Assert.assertEquals
+import org.junit.Before
 import org.junit.Test
-import org.junit.runner.RunWith
-import org.mockito.junit.MockitoJUnitRunner
+import java.lang.reflect.Field
+import java.lang.reflect.Modifier
 
-@RunWith(MockitoJUnitRunner::class)
 class CardDrawerFontTest {
 
+    @Before
+    fun setUp() {
+        val typeface = mockk<Typeface>(relaxed = true)
+        setFinalStatic(Typeface::class.java.getField("MONOSPACE"), typeface)
+    }
+
+    private fun setFinalStatic(field: Field, newValue: Any?) {
```

Comment:
I ended up forgetting to suggest it in our conversation, but this setFinalStatic can be reused at other times. What do you think we create a test extension with it ? It will make your test cleaner and we will be able to reuse this method. What do you think ?

Reply:
Adjustment performed in the TestUtils class